pkg/util/deployment/deployment.go

```diff
@@ -25,9 +25,11 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 	"k8s.io/kubernetes/pkg/apis/extensions"
 	clientset "k8s.io/kubernetes/pkg/client/clientset_generated/internalclientset"
+	extensions_unversioned "k8s.io/kubernetes/pkg/client/typed/generated/extensions/unversioned"
```

Comment:
IIRC, we should not use underline in imported package names. see https://github.com/kubernetes/kubernetes/issues/15319